lib/blockchain/tyronzil.ts

```diff
@@ -429,46 +429,84 @@ export default class TyronZIL extends ZilliqaInit {
         guardianship: TransitionValue,
         id: string,
         amount: string,
-        tyron: TransitionValue
+        addr: string,
+        dID: string,
+        tyron: TransitionValue,
+        doc: any
```

Comment:
hi @DevPH-Wistkey, let's send only the version instead of the full doc.

Reply:
Updated @tralkan